chart/values.yaml

```diff
@@ -6,6 +6,22 @@ image_repository: quay.io
 image_name: pet-battle-tournament
 image_namespace: eformat
```

Comment:
Wrong namespace ?

Reply:
this is the namesapce for the image registry e.g. quay.io/eformat

i have just created for us this:

https://quay.io/organization/petbattle

the idea here is helm does not care where the image sits - so you override these values to be this in openshift:

image-registry.openshift-image-registry.svc:5000/namespace

these are just defaults .. i will add us into the quay.io/petbattle repo above .. and we can go from there? yeah